### ISSUE TITLE
The variable naming is too odd to theme

### ISSUE THREAD

**Issue by condy0919** [rswgnu/hyperbole#44]
The variables/functions have the different prefixes, it looks like too odd in today.

FYI https://github.com/emacscollective/no-littering/pull/146

**Comment by rswgnu** [comment]
Yes, that is a product of time and will take some time to resolve but we wiuld like to unify the prefixes sometime.
